include/xtensor/xfixed.hpp

```diff
@@ -619,19 +633,23 @@ namespace xt
 
     /**
      * Allocates an xfixed_container with shape S with values from a C array.
-     * The type returned by get_init_type_t is raw C array ``value_type[X][Y][Z]`` for ``xt::xshape<X, Y, Z>``.
-     * C arrays can be initialized with the initializer list syntax, but the size is checked at compile
+     * The type returned by get_init_type_t is raw C array ``value_type[X][Y][Z]`` for ``xt::xshape<X, Y,
+     * Z>``. C arrays can be initialized with the initializer list syntax, but the size is checked at compile
      * time to prevent errors.
```

Comment:
Manual correction:
```suggestion
     * The type returned by get_init_type_t is raw C array ``value_type[X][Y][Z]`` for
     * ``xt::xshape<X, Y, Z>``. C arrays can be initialized with the initializer list syntax,
     * but the size is checked at compile time to prevent errors.
```